.circleci/config.yml

```diff
@@ -0,0 +1,86 @@
+---
+version: 2.1
+
+docker-auth: &docker-auth
+  auth:
+    username: $DOCKERHUB_USERNAME
+    password: $DOCKERHUB_PASSWORD
+
+jobs:
+  build:
+    docker:
+      - image: golang:1.13-alpine
+        <<: *docker-auth
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - go-mod-v1-{{ .Branch }}-{{ checksum "go.sum" }}
+            - go-mod-v1-{{ .Branch }}-
+            - go-mod-v1-
+      - run:
+          name: Install dependencies
+          command: |
+            apk add -U zip make
+      - run:
+          name: Build Go package
+          command: |
+            VERSION=$(grep ecs-goploy cmd/version.go | awk -F\" '{print $2}' | awk '{print $2}')
+            make linux VERSION=$VERSION
+      - save_cache:
+          key: go-mod-v1-{{ .Branch }}-{{ checksum "go.sum" }}
+          paths:
+            - "/go/pkg/mod"
+      - persist_to_workspace:
+          root: ./
+          paths:
+            - "packages/*"
+
+  publish:
+    docker:
+      - image: golang:1.13-alpine
```

Comment:
consider using plain alpine image here, as it looks like golang is not needed in this job

Reply:
You are right, we don't need Golang here. Though, we still need Python for AWS CLI, so I replaced it with python:alpine image. Thanks!